src/tests/keywords/pad

```diff
@@ -9,35 +9,35 @@ update request {
 #
 #	rpad tests
 #
-if ("%{rpad:&Tmp-String-0 7}" != "test   ") {
+if ("%{rpad: 7 %{Tmp-String-0}}" != "test   ") {
```

Comment:
Why change the argument order? Padding with spaces like this looks really weird (and to be technically correct, it should have two space at the start - the pad and the separator - which is even weirder).